server/manifest.go

```diff
@@ -18,6 +18,7 @@ type Manifest struct {
 	MediaType     string   `json:"mediaType"`
 	Config        *Layer   `json:"config"`
 	Layers        []*Layer `json:"layers"`
+	Ollama        string   `json:"ollama"`
```

Comment:
if using `masterminds/semver`, this should be `*semver.Version` instead of string with omitempty